### DISCUSSION TITLE
no database name and table name on the dashboard

### DISCUSSION THREAD

**Discussion by YULINHEEE** [pawelsalawa/sqlitestudio#4954] [General]
### Details
Hi friends. I met a weird display problem of SQLite studio. There is no database and table name on the side. I have tried every option about configuration, and uninstall and reinstall several times，but still doesn't work.  There is no bug reports at all.
<img width="393" alt="微信图片_20240313212040" src="https://github.com/pawelsalawa/sqlitestudio/assets/162960421/d8ac6ddd-7c0d-4bad-899c-c022c81d0670">

I use SQLiteStudio version 3.4.4  

Has anyone encountered similar problems to me? Hope to get a solution. Thanks soooo much.

**Comment by pawelsalawa** [comment]
Try going to configuration -> look & feel -> colors -> click "Reset to defaults".

**Comment by YULINHEEE** [comment]
> Try going to configuration -> look & feel -> colors -> click "Reset to defaults".
Thanks so much for your suggestions which make me more certain that this is a configuration problem. I find the problem is the database list front is "<".  Now it works well. Thanks again!
